docs/design.md

```diff
@@ -629,27 +629,50 @@ There are some additional system entries sprinkled amongst the
 non-system keys. See the Key-Prefix Accounting section in this document
 for further details.
 
-# Node Storage
-
-Nodes maintain a separate instance of RocksDB for each disk. Each
-RocksDB instance hosts any number of ranges. RPCs arriving at a
-RoachNode are multiplexed based on the disk name to the appropriate
-RocksDB instance. A single instance per disk is used to avoid
-contention. If every range maintained its own RocksDB, global management
-of available cache memory would be impossible and writers for each range
-would compete for non-contiguous writes to multiple RocksDB logs.
-
-In addition to the key/value pairs of the range itself, various range
-metadata is maintained.
-
--   participating replicas
-
--   consensus metadata
-
--   split/merge activity
-
-A really good reference on tuning Linux installations with RocksDB is
-[here](http://docs.basho.com/riak/latest/ops/advanced/backends/leveldb/).
+# Stores and Storage
+
+Nodes contain one ore more stores. And each of those stores should be on a
+unique disk and contains an instance of RocksDB. And these stores in turn have
+a collection of range replicas. More than one replica for a range will never
+occur on the same store or even the same node.
+
+Early on, when a cluster is first initialized, the few default starting ranges
+will only have a single replica, but as soon as other nodes are available they
+will replicate to them until they've reached their desired replication factor,
+the default being 3. Ranges can have different replication factors and when set
+they will up or down replicate to the appropriate number of replicas. Since
+ranges are only created via splits, all replicas for a range split at the same
+time so the replication factors rarely need to be adjusted.
```

Comment:
I'm not sure how the first part of the sentence connects to the second. Why does the splitting of a range to create new ranges affect the desire to change replication factors?